src/domain/app/i18n/sv.json

```diff
@@ -710,7 +710,8 @@
     "userMenu": {
       "ariaLabelButton": "Öppna användarmenyn",
       "logout": "Logga ut",
-      "openMyProfile": "Min profil"
+      "openMyProfile": "Min profil",
+      "openHelsinkiProfile": "Helsingfors Profil"
```

Comment:
```suggestion
      "openHelsinkiProfile": "Helsingforsprofilen"
```
Because on https://profiili.hel.fi/login using Swedish language:
![image](https://github.com/user-attachments/assets/f96fcb93-d59f-4e08-8a91-8bad8e030fb8)